_articles/manage-repos.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: article
+title: Manage Sources in Pop!_OS
+description: >
+  Add, remove and edit repositories in Pop!_OS
+keywords:
+  - Pop!_OS 18.04
+  - Pop!_OS 19.10
+  - repoman
+  - System76
+  - apt
+image: http://support.system76.com/images/pop-icon.png
+hidden: false
+section: pop-ubuntu
+
+---
+
+### Accessing Repoman
+
+The tool that Pop!_OS has for adding, removing and editing repositories is called <u>Repoman</u> which is a part of the Pop!_Shop.
+
+![Pop!_OS Shop](/images/manage-repos/Pop!_Shop.png)
+
+Once <u>Repoman</u> is opened click on the **Extra Sources** button from this tab there will be options to add repositories with the **Plus** button on the bottom left and edit repositories with the **Edit** button next to the **Plus** button once a repository is selected.
```

Comment:
Once Repoman is opened[comma] click on the Extra Sources button. From this tab, click on the plus button (on the bottom left) to add repositories. Select a repository from the list and then click on the Edit button (next to the Plus button) to edit existing repository lists.